utils/outputwriter/standardoutput.go

```diff
@@ -5,7 +5,6 @@ import (
 	"strings"
 
 	"github.com/jfrog/froggit-go/vcsutils"
-	"github.com/jfrog/jfrog-cli-core/v2/xray/formats"
 )
 
 type StandardOutput struct {
```

Comment:
StandardOutput is not the best name IMO.
I suggest renaming it to "DecoratedOutput".

Reply:
@yahavi, I believe "StandardOutput" is a more fitting term. This is a Markdown interface, and "Standard" implies that it adheres to all the available features in Markdown, essentially representing the expected output. So, in my view, it's not just a decoration but rather the standard output in Markdown.